howto/upgrade/upgrade-anbox.md

```diff
@@ -151,8 +155,50 @@ In case a reboot of the machine is required, a status message will be shown. Whe
     juju run --wait=1m lxd/0 clear-notification
 
 If the LXD charm is deployed on a machine with an NVIDIA GPU installed, by default, the NVIDIA drivers are held from being upgraded in case of downtime for all running instances due to either a manual upgrade or an [unattended-upgrade](https://wiki.debian.org/UnattendedUpgrades). The downside to this is that the machine may miss security updates for the NVIDIA drivers. To manually upgrade the NVIDIA drivers, you need to run the following Juju action:
-    
+
     juju run --wait=30m lxd/0 upgrade-gpu-drivers
+</details>
+
+```{important}
+For users running LXD clusters with the LXD snap tracking a channel which is different than 5.21/stable, it is important that you set the charm configuration item `channel` *explicitly* to the currently running channel before running the `upgrade-cluster` command. This can be done using:
+    juju config lxd channel=5.21/stable
```

Comment:
As mentioned above, can we add a bit more context about it E.g. `If the current LXD cluster consists of the LXD snap tracking from 5.0 channel, it is important that you set.... This can be done using:
    juju config lxd channel=5.0/stable
`